src/app/utils/constants.ts

```diff
@@ -108,6 +108,7 @@ export class Constants {
   };
 
   public static readonly statuses = [
+    'ABANDONED', // may not be an actual status
```

Comment:
What are we unsure of in regards to this status?  Is it something that comes from Tantalis (I thought abandoned was a made up internal status that wraps a couple real tantalis statuses)?